Source/cmake/OptionsWPE.cmake

```diff
@@ -202,6 +203,12 @@ set(LIB_INSTALL_DIR "${CMAKE_INSTALL_FULL_LIBDIR}" CACHE PATH "Absolute path to
 set(EXEC_INSTALL_DIR "${CMAKE_INSTALL_FULL_BINDIR}" CACHE PATH "Absolute path to executable installation directory")
 set(LIBEXEC_INSTALL_DIR "${CMAKE_INSTALL_FULL_LIBEXECDIR}/wpe-webkit-${WPE_API_VERSION}" CACHE PATH "Absolute path to install executables executed by the library")
 
+if (ENABLE_WPE_PLATFORM)
+  SET_AND_EXPOSE_TO_BUILD(ENABLE_DAMAGE_TRACKING ON)
+else ()
+  SET_AND_EXPOSE_TO_BUILD(ENABLE_DAMAGE_TRACKING OFF)
```

Comment:
This will not allow you to disable the option at build time for WPE, because you are unconditionally setting if here depending on the value of `ENABLE_WPE_PLATFORM`. What you want to do is leave it by default as `ON` when defining the option, and then using `WEBKIT_OPTION_DEPEND(ENABLE_DAMAGE_TRACKING ENABLE_DAMAGE_TRACKING)`.